data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,172 @@
+				accept
+	npc accompany
+		government "Remnant (Research)"
+		personality launching timid escort
+		planet "Far Monad"
+		ship "Dromedary Wreck" "Shattered Light"
+	npc evade
+		government "Korath"
+		personality entering plunders harvests
+		system Lucina
+		fleet "Korath Ember Waste Raid" 1
+	on complete
+		payment 250000
+		event "remnant: shattered light"
+		clear "Remnant: Shattered Light 3b: failed"
+		set "Remnant: Shattered Light 3b: done"
+		conversation
+			`	The Shattered Light is so big that traffic control directs the two of you to a large valley next to the spaceport, where you discover that a section of chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of freight, this area looks like it is predominantly used by bulk freight shipments and little else. Now, however, as the Shattered Light slowly moves in, a smattering of Pelicans scatter in front of it looking like nothing more than a flock of seagulls around a camel.`
+			`	Down on the ground, you see Ruach and her team exit an airlock on the side of the Shattered Light and jump down to the ground. They gather around, giving hugs. "<first>!" exclaims Ruach. "We have recovered a lost piece of Remnant history. Thank you for helping. I do not know what this will mean for us in the long term, but it will be a lot of careful examination for the next few weeks."`
+			choice
+				`	"I have things to do, so just let me know if I can help."`
+					goto shatteredhelp
+				`	"So, what happened to it?"`
+			`	"We have had the opportunity to start querying the onboard systems and piece together a little of the Shattered Light's history. As we all know, it was part of the exodus fleet that entered the Terminus wormhole. Unlike most of the other ships, however, it never emerged from the other side. Fortunately, it had a significantly larger array of instruments than was normal for ships of its type, and it looks like at least some of the databanks survived."`
+			`	At this point another Remnant takes up the narrative. "Based on the initial incident logs, it appears that something in the stabilization fields failed during transit. The data is not clear whether the systems failed, or if something in the wormhole fluctuations exceeded the parameters that the systems could handle. In any case, the ship appears to have been shunted laterally from the wormhole in mid-transit.`
+			`	"It is unclear what happened after that, but the ship was subject to significant forces including unidentified impacts that caused multiple system failures. The data loses some coherence at this point, but it basically seems to have been semi-permanently stuck out-of-phase with prime weft until an energy surge destabilized it and it wobbled back into prime coherence. At that point, it started being visible again, and shortly after you spotted it."`
```

Comment:
> stuck **out-of-phase** with **prime weft** until an energy surge destabilized it and it **wobbled back** into **prime coherence**

No idea what those words mean in context. How FW introduces hyperlane creating technology (in a way) is how I think you should do this as well. I.E. using simple concepts.

Reply:
Part of the intent here is to actually create a sort of jarring scenario. These are excited scientists who just did something awesome, and they're forgetting to simplify things as a result.